pymc4/inference/utils.py

```diff
@@ -1,13 +1,14 @@
 from typing import Optional
+import numpy as np
 
-from pymc4 import Model, flow
+from .. import Model, flow
 
 
 def initialize_state(model: Model, observed: Optional[dict] = None) -> flow.SamplingState:
```

Comment:
Maybe refine the type declaration here?  I.e., change to `Optional[Dict[x, y]]` for some *x* and *y*?  Or declare a type for this kind of dictionary, e.g., `ObsDict = Dict[x, y]`?

Reply:
There is nothing special about observed except keys are strings. The API is not that narrow at this point.

Reply:
I think that would be `Dict[str, Any]` then.